test/units/testsuite-21.sh

```diff
@@ -33,17 +33,25 @@ BUS_LIST=(
     org.freedesktop.locale1
     org.freedesktop.login1
     org.freedesktop.machine1
-    org.freedesktop.network1
     org.freedesktop.portable1
     org.freedesktop.resolve1
     org.freedesktop.systemd1
     org.freedesktop.timedate1
-    org.freedesktop.timesync1
 )
 
 # systemd-oomd requires PSI
 if tail -n +1 /proc/pressure/{cpu,io,memory}; then
-    BUS_LIST+=(org.freedesktop.oom1)
+    BUS_LIST+=(
+        org.freedesktop.oom1
+    )
+fi
+
+# systemd-networkd and systemd-timesyncd don't work in containers
```

Comment:
`networkd` can be run in containers if it has `CAP_NET_ADMIN` there. I think it should be skipped in containers if it can't be activated with `busctl introspect org.freedesktop.network1 /`. Something like
```
if ! systemd-detect-virt --container; then
    BUS_LIST+=(
        org.freedesktop.network1
        org.freedesktop.timesync1
    )
elif busctl introspect org.freedesktop.network1 / >&/dev/null then
    BUS_LIST+=(org.freedesktop.network1)
fi
```
should do I think.

Reply:
Ah, interesting, I thought that running systemd-networkd in a container was impossible. Let me tweak that, thanks!